README.md

```diff
@@ -8,22 +8,31 @@ The role installs PostgreSQL, GlassFish and other prerequisites, then deploys Da
 Running the following commands as root should install the latest released version of Dataverse.
 
 	$ git clone https://github.com/IQSS/dataverse-ansible.git dataverse
-	$ ansible-playbook --connection=local -v -i dataverse/inventory dataverse/dataverse.pb -e dataverse/defaults/main.yml
+	$ ansible-playbook --connection=local -v -i dataverse/inventory dataverse/dataverse.pb -e @dataverse/defaults/main.yml
 
 ## Configuration
 
 Installation, customization, administration, and API documentation can be found in the [Dataverse 4 Guides](http://guides.dataverse.org/en/latest/).
 
 The preparation lies in the group_var options (usernames/passwords, whether to install Shibboleth, etc.).  Your \<group_vars_file> may be a set of generic defaults stored in [roles/dataverse/defaults/main.yml](roles/dataverse/defaults/main.yml), but you'll likely want to modify this file or copy it and edit to suit your needs.  Then, fire away:
 
-### Usage:
+### Full(er) Usage:
 	$ git clone https://github.com/IQSS/dataverse-ansible.git dataverse
 	$ export ANSIBLE_ROLES_PATH=.
 	$ ansible-playbook -i <inventory file> [-u <user>] [-b] [-K] -e @<group_vars_file> [-v] dataverse.pb
 
++--------+-----------------------------------------+----------+
+| option | expansion                               | required |
++========+=========================================+==========+
+| -b     | <B>ecome                                | yes      |
+| -K     | as<K> for elevelated privilege password | yes      |
+| -e     | <E>xtra variables file                  | no       |
+| -v     | run with <V>erbosity (up to three Vs)   | no       |
++--------+-----------------------------------------+----------+
```

Comment:
This table looks weird. Here's a screenshot from https://github.com/IQSS/dataverse-ansible/blob/65aa77a4f7893b7ed3fd151779e02e67684be350/README.md#fuller-usage

![Screen Shot 2019-06-25 at 3 29 08 PM](https://user-images.githubusercontent.com/21006/60127362-fd5d1780-975d-11e9-8d96-64b8d9fa6038.png)

Tables in Markdown are hard. That's why I never use them. 😄